src/components/loginform.jsx

```diff
@@ -71,14 +69,18 @@ const LoginForm = () => {
 
     await loginUser();
   };
-
-  // Function to perform the login action
+  const base64Encode = (str) => {
+    return btoa(unescape(encodeURIComponent(str)));
+  };
+  
   const loginUser = async () => {
+    const encodedPassword = base64Encode(password);
+  
     const credentials = {
       usernameOrPhoneNumber: username,
-      password,
+      password: password,  // Use the Base64 encoded password
     };
-
+  
     try {
       const response = await fetch('http://localhost:8080/api/login', {
```

Comment:
Stick to using either Fetch or Axios , maintain consistency throughout the project.